src/main/java/com/bina/varsim/util/Position1.java

```diff
@@ -0,0 +1,20 @@
+package com.bina.varsim.util;
```

Comment:
why do you need two classes? can't you just have a parameter which indicates the indexing scheme?